bin/split_vcf_by_variants.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import csv
+import os
+from typing import Optional
+
+
+def determine_split_size(inputFile, size):
+    with open(inputFile, "r") as variants:
+        input_lines = variants.readlines()
```

Comment:
Cosmetics: maybe use camelCase consistently

Reply:
I would stick to snake case for functions, variables here, just forgot to change some of them. I will fix that 👍